### ISSUE TITLE
Wrong value for alertmanager tls.key

### ISSUE THREAD

**Issue by abullock66** [sassoftware/viya4-deployment#280]
I noticed in a 2022.1.3 deployment with monitoring and logging enabled, that the tls.key in the alertmanager-ingress-tls-secret was incorrect. The certificate was added instead of the key.

The following messages are logged in the ingress-nginx-controller log:

```
Error obtaining X.509 certificate: no object matching key "monitoring/alertmanager-ingress-tls-secret" in local store
Error getting SSL certificate "monitoring/alertmanager-ingress-tls-secret": local SSL certificate monitoring/alertmanager-ingress-tls-secret was not found. Using default certificate
```

It looks like like the issue is on line 124 in the https://github.com/sassoftware/viya4-deployment/blob/main/roles/monitoring/tasks/cluster-monitoring.yaml file

```
        tls.crt: >-
          {{ lookup('file', V4M_ALERTMANAGER_CERT ) |b64encode }}
        tls.key: >-
          {{ lookup('file', V4M_ALERTMANAGER_CERT ) |b64encode }}
      type: kubernetes.io/tls
```
The cert is used instead of the key

**Comment by sayeun** [comment]
This problem has been fixed and the code was merged in staging under https://github.com/sassoftware/viya4-deployment/pull/265.  Will go to production soon.

**Comment by jarpat** [comment]
fixed by #265
